.github/workflows/tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+  schedule:
+    - cron: '0 0 * * sun,wed'
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {name: Linux, python: 3.6, os: ubuntu-16.04, tox: 'flake8,flake8-docs'}
```

Comment:
Should we use a more recent Ubuntu version?

Reply:
I followed the existing tests in .travis.yml. I can update to use 'ubuntu-latest' for everything.

Reply:
I think 20.04 is the way to go here.